src/main/modules/document-download/index.ts

```diff
@@ -45,6 +45,24 @@ export class DocumentDownloadMiddleware {
       },
     };
 
+    const dmStoreProxyForDeemedAsServicePdf = {
+      endpoints: ['/downloads/certificate-of-deemed-as-service'],
```

Comment:
perhaps we could  have an index like: 
```
{
  '/downloads/certificate-of-deemed-as-service': DocumentType.DEEMED_AS_SERVICE_GRANTED,
  '/downloads/certificate-of-dispense-with-service': DocumentType.DISPENSE_WITH_SERVICE_GRANTED
}
```

the loop over Object.entries() and set each one up